### PR TITLE
2211.release-1.0.8

### PR DESCRIPTION
**Pull request by daira**
Release 1.0.8. refs #2211

### CODE REVIEW

File: doc/man/zcash-cli.1 [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+.TH ZCASH-CLI "1" "March 2017" "zcash-cli v1.0.8" "User Commands"
 .SH NAME
-zcash-cli \- manual page for zcash-cli v1.0.7
+zcash-cli \- manual page for zcash-cli v1.0.8
 .SH DESCRIPTION
-Zcash RPC client version v1.0.7-1
+Zcash RPC client version v1.0.8
+.PP
+In order to ensure you are adequately protecting your privacy when using Zcash,
+please see <https://z.cash/support/security/index.html>.
```
Comment (str4d) [inline doc/man/zcash-cli.1]: Nice side-effect of #2203 :smile:

**Comment by daira** [comment]
@zkbot r+

**Comment by daira** [comment]
@ageis wrote:
> AMIs updated and tests can be re-run.

@zkbot retry

**Comment by daira** [comment]
@zkbot retry

**Comment by ageis** [comment]
Test apparently failed due to some packages from Debian sid still being installed on the jessie-based buildslaves. Downgraded those, now @zkbot retry

**Pull request merged**